lib/fs.js

```diff
@@ -113,9 +118,7 @@ function maybeCallback(cb) {
 function makeCallback(cb) {
   if (cb === undefined) {
     return rethrow();
-  }
-
-  if (typeof cb !== 'function') {
+  } else if (typeof cb !== 'function') {
     throw new TypeError('"callback" argument must be a function');
```

Comment:
@yorkie This `callback` actually corresponds to the asynchronous function parameter, not this function's parameter.